conjureup/controllers/credentials/common.py

```diff
@@ -17,4 +18,17 @@ def __init__(self):
 
     def finish(self, cred):
         app.current_credential = cred
+
+        # We need to also attempt to login to the provider as we dont
+        # call save_credential.
+        # XXX: is this the best approach?
```

Comment:
@johnsca im not sure I particularly like this approach but I couldnt come up with a more elegant way since each provider tends to provide different attributes.